.github/workflows/dockerfile-output-appdir

```diff
@@ -0,0 +1,72 @@
+# This dockerfile builds Neovim into an AppImage "AppDir".
+#
+# If you run docker build with the `-o <dir>` flag, it will copy the contents of
+# the final stage out to that dir.
+#
+# NOTE: This *does not* export an AppImage, it exports data to create one!
```

Comment:
even so, it would help discoverability if the file is named `dockerfile-output-appimage`, imo.

Reply:
Renamed.